python/pyarrow/_compute.pyx

```diff
@@ -690,99 +677,96 @@ class CastOptions(_CastOptions):
 
 
 cdef class _ElementWiseAggregateOptions(FunctionOptions):
-    def _set_options(self, bint skip_nulls):
+    def _set_options(self, skip_nulls):
         self.wrapped.reset(new CElementWiseAggregateOptions(skip_nulls))
 
 
 class ElementWiseAggregateOptions(_ElementWiseAggregateOptions):
-    def __init__(self, bint skip_nulls=True):
+    def __init__(self, skip_nulls=True):
```

Comment:
Should this one become keyword-only? `ElementWiseAggregateOptions(true)` isn't very explicit.